internal/gxexec/execute.go

```diff
@@ -0,0 +1,67 @@
+//go:build gx
+
+package gxexec
+
+import (
+	"strings"
+
+	"github.com/janpfeifer/gonb/common"
+	"github.com/janpfeifer/gonb/internal/goexec"
+	"github.com/janpfeifer/gonb/internal/kernel"
+
+	"github.com/janpfeifer/gonb/internal/jpyexec"
+	"k8s.io/klog/v2"
+)
+
+// Executes the %%gx cell magic command.
+//
+// Supports compiling and generating binding for gx functions
+// so they can be used from Go code in the notebook.
+func ExecuteCell(msg kernel.Message, goExec *goexec.State, lines []string) error {
+	if klog.V(2).Enabled() {
+		klog.Infof("GX cell: %q", strings.Join(lines, "\n"))
+	}
+
+	stdout := kernel.NewJupyterStreamWriter(msg, kernel.StreamStdout)
+
+	// Find the last line that is gx code, ie before we see %go or %%
+	last_gx_line := -1
+	for i, line := range lines {
+		if line == "%go" || line == "%%" {
```

Comment:
Since we are using "%%gx" to start GX (as with "%%bash"). What about we use "%%go" as the escape back to Go ?